src/api-calls/api-calls.js

```diff
@@ -32,3 +32,7 @@ export const postComment = (id, user, value) => {
   };
   return api.post(`/articles/${id}/comments`, commentBody);
 };
+
+export const deleteComment = (commentId) => {
+  return api.delete(`/comments/${commentId}`);
+};
```

Comment:
Good job extracting this and yeah there's no point trying to access anything in a .then